source/blood/src/controls.cpp

```diff
@@ -515,12 +515,11 @@ void ctrlGetInput(void)
         if (info.mousey == 0)
         {
             if (gMouseAim)
-                input.q16mlook = fix16_sadd(input.q16mlook, fix16_sdiv(fix16_from_int(info.dpitch>>4), F16(128)));
+                input.q16mlook = fix16_sadd(input.q16mlook, fix16_sdiv(fix16_from_float(scaleAdjustmentToInterval(info.dpitch)/16.f), F16(128)));
             else
-                input.forward -= info.dpitch>>1;
+                input.forward -= int(scaleAdjustmentToInterval(info.dpitch)/2.f);
         }
-        if (input.q16turn == 0)
-            input.q16turn = fix16_sadd(input.q16mlook, fix16_sdiv(fix16_from_int(info.dyaw>>4), F16(32)));
+        input.q16turn = fix16_sadd(input.q16turn, fix16_sdiv(fix16_from_float(scaleAdjustmentToInterval(info.dyaw)/16.f), F16(32)));
```

Comment:
I want to double check that the change on this line from `q16mlook` to `q16turn` is intentional and correct. It certainly looks better this way.